sql/database.go

```diff
@@ -61,29 +62,27 @@ type Encoder func(*Statement)
 type Decoder func(*Statement) bool
 
 func defaultConf() *conf {
-	migrations, err := StateMigrations()
-	if err != nil {
-		panic(err)
-	}
-
 	return &conf{
-		connections:   16,
-		migrations:    migrations,
-		skipMigration: map[int]struct{}{},
-		logger:        zap.NewNop(),
+		enableMigrations: true,
+		connections:      16,
+		logger:           zap.NewNop(),
+		schema:           &Schema{},
 	}
 }
 
 type conf struct {
-	flags         sqlite.OpenFlags
-	connections   int
-	skipMigration map[int]struct{}
-	vacuumState   int
-	migrations    []Migration
-	enableLatency bool
-	cache         bool
-	cacheSizes    map[QueryCacheKind]int
-	logger        *zap.Logger
+	enableMigrations  bool
+	forceFresh        bool
+	forceMigrations   bool
+	connections       int
+	vacuumState       int
+	enableLatency     bool
+	cache             bool
+	cacheSizes        map[QueryCacheKind]int
+	logger            *zap.Logger
+	schema            *Schema
+	allowSchemaDrift  bool
+	ignoreSchemaDrift bool
```

Comment:
It's confusing there are two fields that essentially mean the same thing. How about:
```suggestion
	allowSchemaDrift  bool // only useful with checkSchemaDrift = true
	checkSchemaDrift bool
```

Reply:
Fixed